zip-0401.rst

```diff
@@ -193,15 +208,25 @@ attacker’s transactions cannot be relied on to expire.
 Deployment
 ==========
 
-This specification is proposed to be implemented in ``zcashd`` v2.1.0. It is
-independent of the Blossom network upgrade.
+This specification was implemented in `zcashd` v2.1.0-1. It is independent of
+the Blossom network upgrade.
+
+The fee threshold for applying the *low_fee_penalty* was reduced from 10000 to
+1000 zatoshis as part of the deployment of ZIP 313 in `zcashd` v4.2.0.
+
+The fee threshold for applying the *low_fee_penalty* is planned to change again
+in `zcashd` v5.5.0 to match the ZIP 317 conventional fee. At the same time, the
```

Comment:
We're going to make this change for our stable release, but I'm not sure which release candidate it will be in
```suggestion
in `zcashd` v5.5.0 and `zebrad` v1.0.0 to match the ZIP 317 conventional fee. At the same time, the
```

Reply:
Does that apply also to the constant changes?

Reply:
Yes, the ZIP 317 conventional fee has been implemented for some time, and the constant changes should be in the next Zebra release candidate, or the one after:
https://github.com/ZcashFoundation/zebra/pull/6521